config.js

```diff
@@ -26,14 +26,14 @@ const config = {
     //       </div>
     //     </a>
     //   </li>`,
-    links: [{ text: '', link: '' }],
-    // search: {
-    //   enabled: true,
-    //   indexName: '',
-    //   algoliaAppId: process.env.GATSBY_ALGOLIA_APP_ID,
-    //   algoliaSearchKey: process.env.GATSBY_ALGOLIA_SEARCH_KEY,
-    //   algoliaAdminKey: process.env.ALGOLIA_ADMIN_KEY,
-    // },
+    links: [{ text: 'CONTACT US', link: 'mailto:mait@plutovr.com' }],
+    search: {
+      enabled: true,
+      indexName: '',
+      algoliaAppId: process.env.GATSBY_ALGOLIA_APP_ID,
+      algoliaSearchKey: process.env.GATSBY_ALGOLIA_SEARCH_KEY,
+      algoliaAdminKey: process.env.ALGOLIA_ADMIN_KEY,
+    },
```

Comment:
Is this doing anything right now?

Reply:
yes! (or rather, will, very soon, Travis-gods willing)
![image](https://user-images.githubusercontent.com/10997806/144940954-61880c17-bf2d-481a-bd54-f58f27f94d0d.png)